File: modulo1/segmento3/apunte6/ejercicio2/ejercicio2.java
import java.util.Scanner;

public class ejercicio2{
	
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);
			
			String salir = "n";			
			
			do{

			System.out.print("Ingrese un numero para la suma: ");
			Integer num1 = keyboard.nextInt();

			System.out.print("Ingrese otro numero para la suma: ");
			Integer num2 = keyboard.nextInt();

			Integer result = num1 + num2;

			System.out.println("El resultado es: " + result);
			
			
			System.out.println("¿Desea realizar otra suma? (s/n) ");
			salir = keyboard.next();
			} while(salir.equals("s"));

	}
}


		

File: modulo1/segmento3/apunte6/ejercicio2/ejercicio2.web/ejercicio2.js
function validar(){
	let continuar = `SI`

	while(continuar==`SI`){

		let num1 = 	parseInt(prompt('Ingrese un numero para la suma: '));
		let num2 = parseInt(prompt('Ingrese otro numero para la suma: '));
		resultado = num1 + num2
		
		alert(`${resultado}`);

		continuar = prompt('Desea continuar, (si/no): ').toUpperCase();
	
	}

}

